README.md

```diff
@@ -1 +1,60 @@
-# bitrise-step-unity-install
\ No newline at end of file
+# Unity Installation Workflow for macOS on Bitrise
```

Comment:
FYI this readme can be automatically generated from the step.yml and docs, here is an example: https://github.com/bitrise-steplib/steps-xcode-test/blob/92d41f39d5b4e6a58f56b77977103c8af0da281d/bitrise.yml#L48